aiobotocore/endpoint.py

```diff
@@ -51,43 +69,43 @@ def __dir__(self):
         return attrs
 
     def __del__(self):
-        self.close()
+        # callers may not read the Body resulting in an unclosed ClientResponse
+        if self._loop.is_running():
+            asyncio.ensure_future(self.__response.release(), loop=self._loop)
```

Comment:
from what I understand release is better than close as close can close the connector.  And close is a coroutine so you need to run it with an ensure_future.  Perhaps then just remove the \__del__?  that way you get the warning